src/test/java/pages/HomePage.java

```diff
@@ -25,6 +26,8 @@ public class HomePage extends BasePage {
     private static final SelenideElement DEPARTMENT_BUTTON = $x("//div[text()=\"Відділення №1 (до 1100 кг): вул. Карбишева, 1 (поруч з Адреналін-Сіті)\"]");
     private static final SelenideElement CHOOSE_DEPARTMENT_BUTTON = $x("//button[@class=\"jsx-1716201709 Button Button_primarySmall\"]");
     private static final ElementsCollection STORE_ELEMENTS = $$x(" //div[@class='jsx-1810534291 FlexList__item']");
+    private static final SelenideElement PLAYMARKET_APP_BUTTON = $x("//span/img[@alt='Google play green']");
+    private static final SelenideElement APPSTORE_APP_BUTTON = $x("//img[@alt='App store green']");
```

Comment:
You can write it like one web element where you give the element name in method
Make it with String.format (%s)